### DISCUSSION TITLE
[Projects Beta] Show number of tasks vs completed task after issue title

### DISCUSSION THREAD

**Discussion by yodahack** [community/community#16616] [Projects and Issues] [answered]
missing this

**Comment by yodahack** [comment, marked as answer]
![issue-view](https://user-images.githubusercontent.com/19958053/168449529-c7213a70-9d9a-4bfc-a5bf-9f924e5afbf3.png)
![project-view](https://user-images.githubusercontent.com/19958053/168449530-4642d702-969d-47be-89e9-590de91e4b6d.png)

**Reply by Malix-Labs** [reply]
_This is not an answer and should probably be moved to [this discussion description](https://github.com/github-community/community/discussions/16616#discussion-4074465) 🙃_

**Comment by EurudytaTeam** [comment]
Yeah, me too!

**Reply by Malix-Labs** [reply]
_This is not an answer 🙃_

**Comment by Malix-Labs** [comment]
#16721 is a duplicate that has more popularity and got [this answer](https://github.com/github-community/community/discussions/16721#discussioncomment-2939031)

**Reply by Neudrino** [reply]
This issue is probably a **duplicate** of the most popular issue:
- :arrow_right: https://github.com/github-community/community/discussions/7575

Other (possible) duplicates are:
- https://github.com/github-community/community/discussions/16616
- https://github.com/github-community/community/discussions/13033
- https://github.com/github-community/community/discussions/16721
- https://github.com/github-community/community/discussions/12050
- https://github.com/github-community/community/discussions/15929

An **answer** from Github as of 2022-06-13 can be found at: https://github.com/github-community/community/discussions/16721#discussioncomment-2939031
